HTML Lesson 4 Shumyakova (index.txt)

LizaShu opened the pull request:
Александр, извините, что кучу коммитов сделала( Не совсем понимаю, что лучше коммитить, какую-то часть текста? или отдельно по тегам?

LizaShu committed:
Added file index.txt

LizaShu committed:
Added tags html, body, head to file index.txt

LizaShu committed:
Added meta title and meta description tags, title tag

LizaShu committed:
Added blockquote and hr tag

LizaShu committed:
Deleted wrong meta tag title

LizaShu committed:
Added h1, div, br tags

LizaShu committed:
Put tag h1 br div infront of blockquote

LizaShu committed:
Added h2,h3,ul, li tags

LizaShu committed:
Added h4,dd,dt,dl,p tags

LizaShu committed:
Added p,h5,h6,i,ol tags

LizaShu committed:
Added p,h5,h6,i,ol tags

LizaShu committed:
Added p,h2 tags and a table

LizaShu committed:
Added sub, span, strong,em, small tags

LizaShu committed:
Minor changes in the 43 line

instinct-it-academy commented on home-4-html-4-intro/index.txt:
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```
![image](https://user-images.githubusercontent.com/29653615/179116117-aefd2104-cd0b-440e-9cec-8dbd8abe3d15.png)
в данном задании не критично, учитывая что заданием было изучение тегов можно и каждый новый зафиксировать. В целом в качестве коммита выступает некоторый законченный кусочек работы, функционала. Учитывая то что задание не сложное и не большое вполне мог быть и один коммит.

instinct-it-academy commented on home-4-html-4-intro/index.txt:
```diff
@@ -0,0 +1,73 @@
+    <meta name="description" content="Узнайте как правильно сортировать мусор и каких ошибок не допускать в этой статье.">
+  </head>
+  <body>
+    </div>
```
![image](https://user-images.githubusercontent.com/29653615/179116304-9117dbc5-c1c3-4f7b-9f3d-cf61570b3e62.png)
не корректно, т.к. сначала тег должен открываться , потом закрываться. Необходимо поменять местами. При это h - уже блочный тег, его можно не оборачивать в div

LizaShu replied:
Исправила теги dev

instinct-it-academy commented on home-4-html-4-intro/index.txt:
```diff
@@ -0,0 +1,73 @@
+		  <tbody>
+            <tr>
+              <th width="100"><strong>Маркировка</strong></th>
+              <td width="100"><strong>Название</strong></td>
```
![image](https://user-images.githubusercontent.com/29653615/179116497-d416ffc7-2f76-4328-9ee6-a69a84e8f862.png)
Если выделили строку под заголовки, было бы хорошо везде использовать th

LizaShu replied:
Заменила теги в строке на th

instinct-it-academy reviewed (changes requested):
Добрый вечер, задание на небольшую доработку. Обратите внимание на комментарии и на форматирование кода - отступы и структура кода. Форматирование не входило в условие задания, но т.к. есть моменты которые необходимо подправить, было бы хорошо уделить и этому немного времени.

LizaShu committed:
Comments to homework 4 html resolved

LizaShu committed:
Comments to homework 4 html resolved

LizaShu committed:
Adding proper formatting

LizaShu committed:
18 07 2022
Merge branch 'main' of https://github.com/LizaShu/html-markup-home-Shumyakova into home-4-html-4-intro

LizaShu committed:
Adding proper formatting to index.txt

LizaShu committed:
Adding final formatting to index.txt
